### PR TITLE
Added @AUTHOR@-Option for Templates

### PR DESCRIPTION
**Pull request by practical-solutions**
This is an option many people would appreciate.

### CODE REVIEW

File: action.php [hunk 1]
```diff
@@ -629,6 +629,9 @@ protected function load_template() {
             '@TPLBASE@' => DOKU_BASE . 'lib/plugins/dw2pdf/tpl/' . $this->tpl . '/',
             '@TPLINC@'  => DOKU_INC . 'lib/plugins/dw2pdf/tpl/' . $this->tpl . '/'
         );
+		
+		global $INFO;
+		$replace['@AUTHOR@'] = $INFO['meta']['contributor'][$INFO['user']];
```
Comment (splitbrain) [inline action.php]: Indentation is wrong.  Also, why isn't this part of the array setup above?

Comment (practical-solutions) [reply action.php]: This is true. I will correct the indentation and add this to the array above.

**Comment by practical-solutions** [comment]
I know that the use of the global $INFO array is also called within a batch export. Also, only the last contributor ist displayed. Both of these aspects were intended for the purposes I have in mind. In my company (a hospital) this is exactly what we need: we use the approve-plugin and want documents to display the last author of a page. Ofcourse this could be extended in the future, but the @AUTHOR@-tag would help us a lot for the beginning. If you agree with this, I would modify the aspects as I commented above and commit these changes.

**Comment by practical-solutions** [comment]
Feature not necessary anymore at this point as the option to add tags through plugins is now included in dw2pdf. Thank you for your comments.

**Pull request closed without merging**